tailwind.config.js

```diff
@@ -45,14 +46,15 @@ module.exports = {
         },
         primary: {
           0: "#FFF",
-          70: "#FFFFFFB2",
+          70: "rgba(255, 255, 255, 0.7)",
           100: "#F7F7F7",
           200: "#F0F0F0",
           300: "#CFD0D0",
           400: "#A0A0A2",
           700: "#323232",
           800: "#1A1A1A",
           900: "#141414",
+          "900-40": "rgba(20, 20, 20, 0.4)",
```

Comment:
We have additional feature like theming. We have to create a pack of css variables and use them in tw config